README.md

```diff
@@ -1,32 +1,33 @@
-lastfmexport
-============
```

Comment:
Nothing's inherently wrong with it, besides perhaps the need to change the number of `=` chars whenever the title changes, resulting in a larger diff and the difference in syntax from deeper headers (h3-h6). It's just what [tidy-markdown](https://github.com/slang800/tidy-markdown) does automatically when it's cleaning up markdown .